hw/ip/otp_ctrl/doc/programmers_guide.md

```diff
@@ -148,7 +148,8 @@ In addition, all unrecoverable OTP `Macro*` errors (codes 0x1, 0x3) trigger a `f
 
 If software receives an `otp_error` interrupt, but all error codes read back as 0x0 (`NoError`), this should be treated as a fatal error condition, and the system should be shut down as soon as possible.
 
-Note that while the `MacroWriteBlankError` is marked as a recoverable error, the affected OTP word may be in an inconsistent state after this error has been returned.
+Note that while the `MacroWriteBlankError` will only be generated if the write attempt over already written data fails within the OTP macro after applying any means supported within it to enable a write on existing data, e.g., a bit-reversal option.
```

Comment:
I think this sentence has got a bit mangled. Do you mean to drop the "while"?

Reply:
Uh, yes. You're totally right. Thanks @rswarbrick .